### PR TITLE
Start server in maintenance mode when deployed

### PR DESCRIPTION
**Pull request by portante**

### CODE REVIEW

**Review by portante — COMMENTED** [review]
Ready for review now that tests are passing.

File: server/bin/unittests [hunk 1]
```diff
@@ -253,6 +253,15 @@ function _verify_output {
     return ${res}
 }
 
+function move_out_of_maint {
+    # Given an mock "/var/www/html", move the target Pbench Server out of
```
Comment (dbutenhof) [inline server/bin/unittests:257]: "a" mock... although calling out "mock" at all seems a bit heavy. E.g., this function would work perfectly well on the real `/var/www/html` or any other directory with the expected subdirectory structure. 🤔

**Review by dbutenhof — APPROVED** [review]

**Review by ndokos — APPROVED** [review]

**Review by webbnh — APPROVED** [review]
👍

**Comment by portante** [comment]
Fixes #1370.

**Pull request merged**